### PR TITLE
[Learn] Added TOC Collapse for smaller devices

### PR DESCRIPTION
**Pull request by NikhilSharma03**
Signed-off-by: Nikhil Sharma <nikhilsharmamusic2000@gmail.com>

**Description**
- Added TOC collapse on learn page for smaller devices to give a better user experience

## Demo

https://user-images.githubusercontent.com/58226527/130319310-8718109a-5f75-451b-b21e-68459edb900e.mp4

**Notes for Reviewers**
- Let me know if there are any other changes we can make.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 267591df847ca644b852556f625d65bf4d01bbfb at: https://6120db22127d4ab300b30d61--layer5ng.netlify.app

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@NikhilSharma03, Could you please decrease the line height between the sub-headings, seems to have increased drastically.

Screenshot 2021-08-21 at 4 28 55 PM

**Comment by NikhilSharma03** [comment]
@warunicorn19 Yeah Sure.

## After Changes

Screenshot from 2021-08-21 18-26-25

**Comment by l5io** [comment]
🚀 Preview for commit 90f8328d759023b6a27ac00ea0d2a3a5588dd141 at: https://6120fc1c5aaa84f0904eb5fa--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
Looks good! Thanks @NikhilSharma03

**Comment by Nikhil-Ladha** [comment]
Looks good @NikhilSharma03 , just one minor change is needed. That is to align the `< Learning Paths` button to the content below it.
Screenshot from 2021-08-21 21-31-22

File: src/components/Learn-Components/TOC-Learning-Path/toc.style.js [hunk 1]
```diff
@@ -77,8 +100,26 @@ const TOCWrapper = styled.div`
             }
         }
     }
-    @media (max-width:767px){
+    
+    @media only screen and (max-width: 991px){
         position: initial;
+        margin-left: .25rem;
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/TOC-Learning-Path/toc.style.js:106]: For the alignment, changing this to `-0.5rem` does the trick on my browser at least. You can give it a try though.

**Comment by warunicorn19** [comment]
> Looks good @NikhilSharma03 , just one minor change is needed. That is to align the `< Learning Paths` button to the content below it.
> Screenshot from 2021-08-21 21-31-22

Wow, extremely sharp eyes😅

**Comment by Nikhil-Ladha** [comment]
You know, with time reviewing UI changes made it compulsory to have sharp eyes.... 😆

File: src/components/Learn-Components/TOC-Learning-Path/toc.style.js (hunk 1 above)
Comment (NikhilSharma03) [reply src/components/Learn-Components/TOC-Learning-Path/toc.style.js]: Got It, It looks good on my browser too. Can you please review @Nikhil-Ladha

**Comment by l5io** [comment]
🚀 Preview for commit 61e1f1191f613b8a3313dedf6038bdbf6643009e at: https://6122491ca243659b43e4c3ec--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
Looking nice and clean.

**Review by Nikhil-Ladha — APPROVED** [review]
Thank you for this PR @NikhilSharma03 🚀

**Pull request merged**